tests/pytorch/test_fusible_ops.py

```diff
@@ -2078,7 +2078,7 @@ def test_backward_activation_bias(
 
         # Check that backward operations have been fused
         backward_ops = model._module_groups[0]._backward_ops
-        if with_quantization and quantization in ["fp8_delayed_scaling", "mxfp8"]:
+        if with_quantization:
```

Comment:
This check could be removed assuming that `BackwardActivationBias` is changed to support the fusion even with no quantization.